code/modules/mob/living/carbon/human/life/handle_mutations_and_radiation.dm

```diff
@@ -73,3 +89,200 @@
+		if(rad_tick > 200)
+			if(prob(2*major_rad_multiplier))
+				//Internal hemorrhaging
+				var/list/limbs_to_bleed = list()
+				for(var/datum/organ/external/E in organs)
+					if(E.is_robotic())
+						continue
+					limbs_to_bleed.Add(E)
+				if(limbs_to_bleed.len)
+					var/datum/organ/external/victim = pick(limbs_to_bleed)
+					if(prob(35))
+						to_chat(src, "<span class = 'danger'>You feel something tear in your [victim.display_name]</span>")
+					var/datum/wound/internal_bleeding/I = new (1*minor_rad_multiplier)
+					victim.wounds += I
+		if(rad_tick > 300)
+			if(prob(5*rad_multiplier))
+				//Organ damage
+				var/list/organ_to_damage = list()
+				for(var/datum/organ/internal/I in internal_organs)
+					if(I.robotic)
+						continue
+					organ_to_damage.Add(I)
+				if(organ_to_damage.len)
+					var/datum/organ/internal/victim = pick(organ_to_damage)
+					victim.damage += rand(1,5)*rad_multiplier
+			if(prob(0.5*major_rad_multiplier))
+				//Become uncloneable
+				if(!(M_NOCLONE in mutations))
+					to_chat(src, "<span class = 'blob'>You feel something twist and break.</span>")
+					mutations |= M_NOCLONE
+		if(rad_tick > 400)
+			if(prob(2*major_rad_multiplier))
+				//Minor limb mutation
+				var/list/datum/organ/external/candidates = list()
+				for (var/datum/organ/external/O in organs)
+					if(!(O.status & ORGAN_MUTATED|ORGAN_ROBOT))
+						candidates.Add(O)
+				if (candidates.len)
+					var/datum/organ/external/O = pick(candidates)
+					O.mutate()
+					to_chat(src, "<span class = 'notice'>Something is not right with your [O.display_name].</span>")
+			if(prob(5*rad_multiplier))
+				//Minor clone damage
+				adjustCloneLoss(rand(1,5))
+				to_chat(src, "<span class='warning'>[pick("You can feel your body becoming weak!</span>", \
+				"You feel like you're about to die!", \
+				"You feel every part of your body screaming in agony!", \
+				"A low, rolling pain passes through your body!", \
+				"Your body feels as if it's falling apart!", \
+				"You feel extremely weak!", \
+				"A sharp, deep pain bathes every inch of your body!")]")
+			if(prob(10)*minor_rad_multiplier)
+				//Blindness
+				var/datum/organ/internal/eyes/E = internal_organs_by_name["eyes"]
+				if(!E.robotic && (sdisabilities & BLIND || disabilities & NEARSIGHTED))
+					to_chat(src, "<span class = 'danger'>[pick("Your eyesight starts to fade!","Your eyes go cloudy!","Are you going blind?")]</span>")
+					E.damage += 2.5
+					eye_blurry = min(eye_blurry+1.5,50)
+					if (E.damage >= E.min_broken_damage && !(sdisabilities & BLIND))
+						simple_message("<span class='warning'>You go blind!</span>","<span class='warning'>Somebody turns the lights off.</span>")
+						sdisabilities |= BLIND
+					else if (E.damage >= E.min_bruised_damage && !(disabilities & NEARSIGHTED))
+						simple_message("<span class='warning'>You go blind!</span>","<span class='warning'>Somebody turns the lights off.</span>")
```

Comment:
The message is the same as for going blind, and the user is not going blind, they're getting another disability. It's confusing.